js/index.js

```diff
@@ -40,3 +40,75 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let navItems = document.querySelectorAll('header nav a');
+let navBar = document.querySelector('nav');
+let homeItem = document.createElement('a');
+let dontClickItem = document.createElement('a');
+dontClickItem.style.color = 'green';
+
+homeItem.textContent = 'Home';
+homeItem.style.color = 'green';
+navItems[0].textContent = 'Services';
+navItems[1].textContent = 'Product';
+navItems[2].textContent = 'Vision';
+navItems[3].textContent = 'Features';
+navItems[4].textContent = 'About';
+navItems[5].textContent = 'Contact';
+dontClickItem.textContent = 'Dont Click';
+navBar.appendChild(dontClickItem);
+navBar.prepend(homeItem);
+
+navItems.forEach(function (ele) {
+  ele.style.color = 'green';
+})
+
+let h1Item = document.querySelector('h1');
+// let br = document.createElement('BR');
+// h1Item.textContent = 'DOM ' + document.write('<br> /') + 'IS AWESOME';
+h1Item.innerHTML = 'DOM <br /> IS <br /> AWESOME';
+let buttonOne = document.querySelector('button');
+buttonOne.textContent = 'Get Started';
+
+let headerIMG = document.getElementById('cta-img');
+headerIMG.src = 'img/header-img.png';
+
+let mainContent = document.querySelectorAll('.text-content');
+mainContent[0].firstChild.textContent = 'Features';
+mainContent[0].lastChild.textContent = 'Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.';
+
```

Comment:
great way to get this done. you could take it one step further by pulling data from the siteContent object.

```
let mainContent = document.querySelectorAll('.text-content');

let cardContent = Object
  .entries(siteContent["main-content"])
  .map(item => item[1]);

mainContent[0].firstChild.textContent = cardContent[0];
mainContent[0].lastChild.textContent = cardContent[1];
```